pkg/southbound/topo/topo.go

```diff
@@ -75,6 +76,11 @@ func ConnectToTopo(ctx context.Context, topoEndpoint string,
 		}
 		switch in.Type {
 		case topodevice.ListResponse_NONE:
+			err := createHandler(in.GetDevice())
```

Comment:
it appears there's a `fallthrough` statement you can use for this. I think I had intended to use comma instead of colon

Reply:
Oh yes you're right. Forgot about that!